sky/cli.py

```diff
@@ -2213,35 +2213,41 @@ def _hint_for_down_spot_controller(controller_name: str):
            f'aware of the following:{colorama.Style.RESET_ALL}'
            '\n * All logs and status information of the spot '
            'jobs (output of `sky spot queue`) will be lost.')
+    if cluster_status == global_user_state.ClusterStatus.INIT:
+        msg += ('\n * If there are pending/in-progress spot jobs, those '
+                'resources will not be terminated and require manual cleanup.')
```

Comment:
Why do we print this only for INIT? Seems applicable to UP as well (what we did before)?

Reply:
In the `UP` cluster, we will directly show what spot jobs are still running or it is safe to terminate the spot controller as there is no spot jobs running.

This message is only for INIT controller where we don't know for sure whether there is a spot job running.

Actually, after thinking more about this, we should ban terminating the controller when it is in INIT mode to avoid potential spot cluster leakage as well.